src/test/scala/chiselTests/CloneModuleSpec.scala

```diff
@@ -0,0 +1,60 @@
+// See LICENSE for license details.
+
+package chiselTests
+
+import chisel3._
+import chisel3.util.{Queue, QueueIO}
+import chisel3.experimental.CloneAndGetIO
+import chisel3.testers.BasicTester
+import org.scalatest._
+import org.scalatest.prop._
+
+class QueueClone extends Module {
+  val io = IO(new QueueIO(UInt(32.W), 2))
+  val q1 = Module(new Queue(UInt(32.W), 2))
+  val q2_io = CloneAndGetIO(q1)
+  q2_io.elements.foreach { case (name, port) => println(name) }
+  q1.io.enq <> io.enq
+  val q2_io_bundle = q2_io("io").asInstanceOf[q1.io.type]
```

Comment:
How does this work with MultiIOModule?

Reply:
It works with MultiIOModule. I added a test.